### DISCUSSION TITLE
Legacy Project Missing

### DISCUSSION THREAD

**Discussion by labrocca** [community/community#139628] [Projects and Issues]
I haven't checked a Project I had on here for months. Now it's 404'd.   It was a very old Project but important because I used the lists for keeping important notes.  Now I've lost access.  Is there any way to recover the Project, export the data, or maybe update it to newest version of Projects?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @labrocca ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
